common/network-common/src/main/java/org/apache/spark/network/util/LevelDBProvider.java

```diff
@@ -134,8 +134,12 @@ public StoreVersion(@JsonProperty("major") int major, @JsonProperty("minor") int
 
     @Override
     public boolean equals(Object o) {
-      if (this == o) return true;
```

Comment:
you know what, these code are generated by IntelliJ...

Reply:
I don't know about these. Sorry to bother you too. Please ignore this PR